test/xspec.bats

```diff
@@ -193,6 +193,10 @@ teardown() {
 
 
 @test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	if [ -z ${XMLCALABASH_VERSION} ]; then
```

Comment:
With this line, to run `xspec.bats` thoroughly, you have to set both `XMLCALABASH_VERSION` and `XMLCALABASH_CP`.
Can you consider checking the existence of `XMLCALABASH_CP` instead of `XMLCALABASH_VERSION`? That makes it a little bit easier to run `xspec.bats` on local console without `.travis.yml`, which I do occasionally.